### DISCUSSION TITLE
Can create SSL cert in AppService using Docker

### DISCUSSION THREAD

**Discussion by carloscds** [rcl-ssl/docs#11] [General]
This app run in a docker container:

Error:Failed to create app service certificate for host : www.xxx.com.br on siteId : 841 with challenge : HTTP. Could not validate www.birdix.com.br with challenge http-01 for domain www.birdix.com.br, status reported : invalid. Invalid response from http://www.xxx.com.br/.well-known/acme-challenge/9FYxEiVp4ngQjJD-Y8wvIj_7L0u7zAru8emCnQlHJe4 [191.233.203.33]: 404 . Recommendation R03: You did not correctly add the validation file to the root of your website, please check that your validation record is available publicy on the web before you validate.. Help link: https://docs.rclapp.com/portal/stand-alone.html#create-a-ssltls-certificate-using-http-01

**Comment by rcladmin** [comment]
Support for app service - web app for containers :

- DNS Challenge - Yes
- HTTP Challenge - No

I will look into providing support for HTTP challenge and see if this is possible.

**Comment by rcladmin** [comment]
Web App for containers will not be supported for the HTTP challenge any time soon. However, the DNS challenge is currently supported.
